CONTRIBUTING.md

```diff
@@ -67,21 +75,36 @@ git submodule update --init --recursive
 apt install libxml2-utils
 ```
 
-#### Launch all tests
+#### Check outputs manually
 
+To validate the output NeTEx obtained it is possible to use xmllint:
```

Comment:
```suggestion
To validate the NeTEx output, it is possible to use xmllint:
```

I'm also wondering if we should add something about the fact that `xmllint` is slow... very slow and on big files it can literally take hours to check.